pkg/api/mlflow/dao/repositories/helpers.go

```diff
@@ -30,3 +32,32 @@ func makeParamConflictPlaceholdersAndValues(params []models.Param) (string, []in
 	}
 	return placeholders, valuesArray
 }
+
+// BuildJsonCondition creates sql and values for where condition to select items having the specified map of json paths
+// and values in the given json column. Json path is expressed as "key" or "outerkey.nestedKey".
+func BuildJsonCondition(
+	dialector string,
+	jsonColumnName string,
+	jsonPathValueMap map[string]string,
+) (sql string, args []any) {
+	if len(jsonPathValueMap) == 0 {
+		return
+	}
+	var conditionTemplate string
+	switch dialector {
+	case postgres.Dialector{}.Name():
+		conditionTemplate = "%s#>>? = ?"
+		for k, v := range jsonPathValueMap {
+			path := strings.ReplaceAll(k, ".", ",")
+			args = append(args, "{"+path+"}", v)
```

Comment:
since we always use `fmt.Sprintf` pattern, will be nice to use it instead of `"{"+path+"}"`

Reply:
Seems more verbose to me, is there some benefit performance-wise?

Reply:
changed it to fmt.Sprintf